### PR TITLE
Updated Readme file with contribution rules

### PR DESCRIPTION
**Pull request by Hristijan95**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,11 @@
...
+To start, you can always check the [open issues](https://github.com/thenewboston-developers/Website/issues) in this repository. You can filter issues using the labels and find the ones that suit your domain. All issues that have a green coins label on them are ready to work on and the amount of coins that is on the "PR Reward" label will be transfered to the account of the person who solves that issue. Please avoid working on issues that have the "Not Ready" label on them.
+
+If you discover a new issue that was not yet reported, please open an issue before working on it. People who discover bugs will be rewarded as well with the amount of coinst specified by the red "Bug Bounty Owed" labels. The reward labels will be assigned on issues by the core team only based on the importance and difficulty of the tasks. If you want to create an issue with a design suggestion, please do it in the [design repository](https://github.com/thenewboston-developers/Design).
```
Comment (jamessspanggg) [inline README.md]: *amount of coins

File: README.md [hunk 2]
```diff
@@ -23,7 +23,11 @@
...
+
+Every PR must be connected to an issue, so as stated above, before rasing a PR, please create an issue and reference it in the PR.
```
Comment (jamessspanggg) [inline README.md:30]: *before raising a PR

File: README.md [hunk 3]
```diff
@@ -23,7 +23,11 @@
 Contributions are what make the open-source community such an amazing place to learn, inspire, and create. Any
 contributions you make are **greatly appreciated**.
 
-To start, you can always check the [open issues](https://github.com/thenewboston-developers/Website/issues) in this repository. You can filter issues using the labels and find the ones that suit your domain. Mostly, you'll see issues labeled with the coins you'll get if you fix that particular issue. Earning coins improves your rank on the [leaderboard](https://thenewboston.com/leaderboard/All).
+To start, you can always check the [open issues](https://github.com/thenewboston-developers/Website/issues) in this repository. You can filter issues using the labels and find the ones that suit your domain. All issues that have a green coins label on them are ready to work on and the amount of coins that is on the "PR Reward" label will be transfered to the account of the person who solves that issue. Please avoid working on issues that have the "Not Ready" label on them.
```
Comment (jamessspanggg) [inline README.md:26]: *transferred

File: README.md [hunk 4]
```diff
@@ -23,7 +23,11 @@
...
+
+If you discover a new issue that was not yet reported, please open an issue before working on it. People who discover bugs will be rewarded as well with the amount of coins specified by the red "Bug Bounty Owed" labels. The reward labels will be assigned on issues by the core team only based on the importance and difficulty of the tasks. If you want to create an issue with a design suggestion, please do it in the [design repository](https://github.com/thenewboston-developers/Design).
```
Comment (jamessspanggg) [inline README.md:28]: *The reward labels will be assigned to issues...

**Comment by lapstjup** [comment]
@Hristijan95 and @thesanjeevsharma  - I had a thought after seeing this. Should we have these steps in a separate **CONTRIBUTING.md** file since that also increases our repo's community profile ?

**Comment by thesanjeevsharma** [comment]
I was thinking the same. Let's merge this and raise an issue for that.

**Comment by Hristijan95** [comment]
Yep, we can do that. I raised this PR to get things going. Thanks, @jamessspanggg for fixing all that. :)

**Pull request merged**